pgcli/key_bindings.py

```diff
@@ -42,7 +42,11 @@ def _(event):
         Force autocompletion at cursor.
         """
         _logger.debug('Detected <Tab> key.')
-        event.cli.current_buffer.complete_next()
+        b = event.cli.current_buffer
+        if b.complete_state:
```

Comment:
The menu is shown when the buffer has a ``complete_state``. If we have a ``complete_state``, then we can simply go to the next completion, otherwise, we call ``cli.start_completion`` which runs the completer asynchronously. (Not blocking the input.)  Probably I need to create function somewhere wrapping around these four lines, because it's a common pattern.